install.md

```diff
@@ -62,19 +39,23 @@ sudo apk add skopeo
 brew install skopeo
 ```
 
+You may need to add registry certificate to your macOS truststore.
+```
+sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <registry_cert.crt>
```

Comment:
This part doesn’t make sense to me.

- AFAICS this has nothing to do with installing Skopeo; just the `brew install` does that.
- WRT configuring a certificate trusted to connect to a registry, platform-agnostic paths like `--cert-dir` are still expected to work
- All of #563 ended up surprising to me. The original report was that that CAs from the macOS keychain were ignored, and the recommended fix is to … add the CA to the macOS keychain? Stated this way, it just does not follow at all, so there must be some subtlety to #563 that wasn’t written down.

Overall I think this part of the PR should just be dropped, and #563 either discussed further, or just closed as “2 years later, this was reported to work fine”.

Reply:
ack, will do. thanks @mtrmac , and apologies if this bothered you on your PTO.

Reply:
(No worries — I chose to take a little time to see what’s new, and I know I had the option not to. But I appreciate the thought all the same.)